Cargo.toml

```diff
@@ -21,4 +21,5 @@ llvm = ["inkwell"]
 structopt = "0.3.21"
 rust-embed = "5.7.0"
 inkwell = { version = "0.1.0-beta.2", features = ["llvm10-0"], optional = true }
-
+regex = "1.5.4"
```

Comment:
I don't think there's a good reason to use regex for this

Reply:
I'm checking for the Unicode `L` category, which contains all letters. Is there a simpler way without regex?

Reply:
There's a Unicode support library: https://github.com/hardboiled65/seshat-unicode

And check can be done like this:

```rust
use seshat::unicode::{Gc, Ucd};

// ...
let category = 'д'.gc();
let is_valid = match category {
    // All L* general categories
    Gc::Ll | Gc::Lm | Gc::Lo | Gc::Lt | Gc::Lu => true,
    _ => false,
}
```

Reply:
Nice. I'll leave this as is and open a followup optimization issue instead.

Reply:
Actually, holy shit. That library is 1.46M in size. Compare that to 237k of Regex. Most of that junk comes from massive embedded Unicode-tables, which is terribly inefficient I think.

Reply:
You can instead take those tables from regex library, and this will be smaller than both of them:

https://github.com/rust-lang/regex/blob/master/regex-syntax/src/unicode_tables/general_category.rs